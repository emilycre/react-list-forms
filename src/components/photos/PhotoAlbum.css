@import url('https://fonts.googleapis.com/css?family=Stylish');

.PhotoAlbum {
  padding: 0.5rem;
  margin-top: 5rem;

  h2 {
    padding: 0.5rem;
    font-family: 'Stylish';
    font-size: 2rem;
    text-align: center;
    text-decoration: none;
  }

  ul {
    display: flex;
    flex-direction: row;
    flex-wrap: wrap;
    list-style-type: none;
  }

  li {
    height: 10rem;
    margin: 1rem;
  }

  img {
    width: auto;
    height: 100%;
    border-radius: 5px;
  }
}
