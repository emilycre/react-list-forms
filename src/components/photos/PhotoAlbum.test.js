import React from 'react';
import { shallow } from 'enzyme';
import PhotoAlbum from './PhotoAlbum';

describe('PhotoAlbum component', () => {
  it('Renders a PhotoAlbum', () => {
    const photos = [
      '/img1',
      '/img2'
    ];
    const wrapper = shallow(<PhotoAlbum title="My Photos" photos={photos} />);
    expect(wrapper).toMatchSnapshot();
  });
});
