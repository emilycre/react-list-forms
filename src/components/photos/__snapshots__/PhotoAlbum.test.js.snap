// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PhotoAlbum component Renders a PhotoAlbum 1`] = `
<section
  className="PhotoAlbum"
>
  <h2>
    My Photos
  </h2>
  <Photos
    photos={
      Array [
        "/img1",
        "/img2",
      ]
    }
  />
</section>
`;
